web/package.json

```diff
@@ -15,10 +15,13 @@
     "react": "^16.8.4",
     "react-app-rewired": "^2.1.0",
     "react-dom": "^16.8.4",
+    "react-json-view": "1.19.1",
+    "react-pager": "^1.3.3",
```

Comment:
it seems react-pager was replaced by rc-pagination

Reply:
oh, I forget . I'm sorry.